.github/workflows/e2e.yml

```diff
@@ -0,0 +1,13 @@
+name: example-cron
+on:
+  schedule:
+    # runs tests every day at 4am
+    - cron: '0 4 * * *'
```

Comment:
Is there a timezone? Is it 4am UTC?

Reply:
Ah true, this is UTC - ends up being midnight; the comment came from a copied comment elsewhere,